src/ui/profile_preferences.ui

```diff
@@ -2636,6 +2636,13 @@ you can use it but there could be issues with aligning columns of text</string>
             </property>
            </widget>
           </item>
+          <item row="3" column="0">
```

Comment:
As this is a checkbox which does not have a separate label it is best to make it span the same space as one of the other label and control pairs (with a `colspan` attribute) and given that there is not anything on `row="2"` it does not need to be `row="3"`: 
```suggestion
          <item row="2" column="0" colspan="2">
```